modules/users/tests/server/user.server.model.tests.js

```diff
@@ -436,17 +440,18 @@ describe('User Model Unit Tests:', function () {
     });
 
     it('should allow valid email address - "abc-def@abc.com"', function (done) {
-      var _user = new User(user);
+      var _user1 = new User(user1);
 
-      _user.email = 'abc-def@abc.com';
-      _user.save(function (err) {
+      _user1.email = 'abc-def@abc.com';
+      _user1.save(function (err) {
         if (!err) {
-          _user.remove(function (err_remove) {
+          _user1.remove(function (err_remove) {
+            should.not.exist(err);
```

Comment:
If the code enters into the code block of 447 then it means that _user1 object was created successfully, and so what we do is take care of removing _user1 object from the db for further tests, and then we assert that (err) object shouldn't exist which is indeed correct because that's the reason we entered the if block in first place.